bedrock/mozorg/templates/mozorg/mpl/2.0/faq.html

```diff
@@ -127,3 +127,7 @@ <h3 id="physical-distribution">Q26: I would like to distribute source code by us
     <p>In modern software development, distribution happens almost exclusively over the internet, because of the low cost and convenience. While the MPL allows for alternatives when necessary, in general any mechanism (such as a courier) that adds cost and complexity without a specific, necessary purpose is not reasonable, and so does not comply with the requirements of Section 3.2(a).</p>
   </section>
 {% endblock %}
+
+{% block structured_data %}
```

Comment:
This template does not have the `structured_data` block yet so the include is not being included.

For this experiment I think it is okay to add just the include inside the block above, after the `</section>`.